src/FakeItEasy/FakeItEasy.csproj

```diff
@@ -32,10 +32,20 @@
   <!-- Release -->
 
   <PropertyGroup Condition="'$(Configuration)' == 'Release'">
-    <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\FakeItEasy.xml</DocumentationFile>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
```

Comment:
Not actually related to the rest PR, but since I was touching that file anyway... I can remove it if you prefer